README.md

```diff
@@ -626,9 +717,58 @@ const contract = new HRC721('0x...00', ABI, wallet)
 const address = await contract.getApproved('1')
 ```
 
-####  totalSupply
+#### setApprovalForAll(addressOperator: string, approved: boolean, txOptions?: ITransactionOptions): Promise&lt;Transaction&gt;
 
-Returns the total amount of tokens stored by the contract.
+<p>Approve or remove <code>addressOperator</code> as an operator for the caller. Operators can call <code>transferFrom</code> or <code>safeTransferFrom</code> for any token owned by the caller.<p>
+<p>Requirements
+  <ul>
+    <li>The <code>addressOperator</code> cannot be the caller.</li>
+  </ul>
+</p>
+<p>Emits an <code>ApprovalForAll</code> event.</p>
+  
+```ts
+import { HttpProvider } from '@harmony-js/network'
+import { PrivateKey, HarmonyShards, HRC721 } from 'harmony-marketplace-sdk'
+import * as ABI from './abi.json'
+
+const wallet = new PrivateKey(
+  new HttpProvider(HarmonyShards.SHARD_0),
```

Comment:
replace this line for HarmonyShards.SHARD_0,

Reply:
just done here: https://github.com/blockcoders/harmony-marketplace-sdk/pull/54/commits/276b5a749d336f82f1f92c221dd287cabe263cf5